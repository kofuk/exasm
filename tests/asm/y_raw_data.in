.word 0xDEAD
.word 0xBEEF
