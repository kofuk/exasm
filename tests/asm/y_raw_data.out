@00 11011110 10101101 // <raw data>
@02 10111110 11101111 // <raw data>
